src/server/migrations/20190918222221_suggested_routes_foreign_key.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+exports.up = function(knex) {
+  return knex.schema.table("suggested_routes", table => {
+    table.dropForeign("voyage_id", "routes_voyage_id_foreign");
+    table
+      .foreign("voyage_id")
+      .references("voyages.id")
+      .onDelete("CASCADE")
+      .onUpdate("RESTRICT");
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.table("suggested_routes", table => {
+    table.dropForeign("voyage_id");
+    table
+      .foreign("voyage_id", "routes_voyage_id_foreign")
+      .references("voyages.id")
+      .onDelete("CASCADE")
+      .onUpdate("RESTRICT");
+  });
+};
```

Comment:
Can you please explain this migration? I don't see it perform any changes to my schema for `suggested_routes`?

Reply:
It just changes foreign key name from routes_voyage_id_foreign to **suggested_**routes_voyage_id_foreign